src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -24,8 +45,23 @@ import { useDolphinPath } from "@/lib/hooks/useSettings";
 
 import { SettingItem } from "./SettingItem";
 
+function TabPanel(props: any) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div role="tabpanel" hidden={value !== index} id={`full-width-tabpanel-${index}`} {...other}>
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
```

Comment:
This typography seems to be giving a bunch of errors in dev tools since it is equivalent to a p element and you can't put some elements inside that like forms.